test/engine/SpatialJoinTestHelpers.h

```diff
@@ -151,4 +151,14 @@ inline std::shared_ptr<QueryExecutionTree> buildSmallChild(
   return buildJoin(qec, scan1, scan2, joinVariable);
 }
 
+inline std::string stringifyVariableToColumnMap(VariableToColumnMap vc) {
+  std::string result = "";
+  auto vec = copySortedByColumnIndex(vc);
+  for (auto [var, info] : vec) {
+    result =
+        absl::StrCat(result, "(", var.name(), " => ", info.columnIndex_, ")");
+  }
+  return result;
+}
+
```

Comment:
You shouldn't use such  {"testing by string" things. setup the expected variable to column map and do `::testing::UnorderedElementsAreArray` or `::testing::ContainerEq`.
(we have  helpers to somewhat elegangly shorten the creation of `ColumnIndexAndtypeInfo `objects, see how other operations test this.

Reply:
When testing with the mentioned matchers we get hex dumps of the contained objects as error messages (which are not very helpful), while with strings we can see and debug a test failure easily. I will change it, but I'm not 100% sure if it is actually better in this case.

Reply:
Ok, so what you actually want is a human-readable serialization of a VariableToColumn Map.
The correct way to implement this is to either implement the `std::ostream& operator<<` for this type or the `PrintTo` function of google test.
You probably also need to implement this for the `ColumnIndexAndTypeInfo` objects.
There are many examples for this in the codebase. Make sure to declare them together with the `VariableToColumnMap`, s.t. ALL tests that use varaible to column maps have the correct info.

Make sure to comment them in a way that tells us that these printers are just for debugging,  and make sure that you add at least some dummy tests for them (mainly that the coverage tools don't complain).

Reply:
Apparently you have already implemented exactly these `PrintTo` functions for the vartocolmap in `test/printers/Variable[ToColumnMap]Printers.h` and I just didn't find them. I have now included those and it works. Therefore my helper function can be removed.